src/Symfony/Component/HttpKernel/EventListener/TranslatorListener.php

```diff
@@ -24,6 +24,8 @@
  * Synchronizes the locale between the request and the translator.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since Symfony 4.3
```

Comment:
a runtime deprecation notice should also be added on top of the file - after the namespace declaration (see existing similar deprecations in the code base)